xml/admin_ceph_upgrade.xml

```diff
@@ -624,6 +624,11 @@ exported keyring for client.bootstrap-osd
      </para>
     </listitem>
    </itemizedlist>
+   <para>
```

Comment:
I think that'd tell the user to upgrade the admin _before_ setting up a salt cluster.
AFAIK the procedure starts with a SP2 based salt cluster, then upgrade the admin to SP3 in order to get the latest code which includes the engulfment

Reply:
If you have a SP2-based salt cluster, and then upgrade the master node to SP3, that becomes a mixed Salt cluster (salt-master and salt-minion on the master node will be from SP3/SES5, and the rest of the minions will be SP2/SES4). That is the starting point for the engulfment and the maintenance.upgrade orchestration that follows.

Reply:
ok, fine with me.